src/client/lobby/room-list.js

```diff
@@ -22,6 +23,16 @@ export default function RoomList(props) {
     );
 }
 
+function getClass(room) {
+    if(room.data.started){
+        return 'list-group-item list-group-item-danger space';
```

Comment:
Copy paste for if else here. The common stuff could just go directly in the component, then you just decide between danger or warning. The function could be overkill...